cmd/transflect-operator/main.go

```diff
@@ -19,6 +21,30 @@ import (
 var (
 	version   = "v0.0.0"
 	errSignal = fmt.Errorf("received shutdown signal")
+
+	// metrics
+	// filtersGauge includes filters that already exist at startup.
+	filtersGauge = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "transflect_envoyfilters",
+		Help: "The total number of EnvoyFilters deleted by transflect",
```

Comment:
Sorry just noticed the help message this time around. Pretty sure this is not "deleted"

Reply:
don't say sorry. thank you for taking sooo much care with this. done.